cmd/lncli/commands.go

```diff
@@ -312,7 +384,7 @@ var CloseChannelCommand = cli.Command{
 	Name: "closechannel",
 	Description: "Close an existing channel. The channel can be closed either " +
 		"cooperatively, or uncooperatively (forced).",
-	Usage: "closechannel funding_txid output_index time_limit allow_force",
+	Usage: "closechannel [Options] funding_txid [output_index [time_limit]]",
```

Comment:
While you're here, can you remove the `time_limit` option from the usage here? It's left over from an old discarded idea. 

Thanks!